packages/core/src/fhir-deduplication/resources/procedure.ts

```diff
@@ -129,9 +140,11 @@ export function extractCodes(concept: CodeableConcept | undefined): {
           cptCode = code;
         } else if (system.includes(LOINC_CODE) || system.includes(LOINC_OID)) {
           loincCode = code;
+        } else if (system.includes(SNOMED_CODE) || system.includes(SNOMED_OID)) {
```

Comment:
If we're adding this, we should also add it to the filter above - [here](https://github.com/metriport/metriport/blob/7a1d068792901e7d0e6441fbaf5b6c972440e019/packages/core/src/fhir-deduplication/resources/procedure.ts#L76-L79)

Reply:
good catch